Add Telnet into docker to test connectivity for cloud getting started (#22283)
gravitational/teleport: docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```

Your terminal will begin a shell session in the `ubuntu:22.10` container. From

> there, make sure you have`curl` and `telnet` installed. Both are used by the install script:

Reviewer: ```suggestion
there, make sure you have `curl` and `telnet` installed. Both are used by the install script:
```

> there, make sure you have `curl` and `telnet` installed. Both are used by the install script:

Reviewer 2: ```suggestion
there, make sure you have `curl` and `telnet` installed. When we run a script to install Teleport later in this guide, the script will require both of these tools.
```
Suggested because this is the first time we mention an installation script—"the install script" doesn't refer to anything here.

```

Your terminal will begin a shell session in the `ubuntu:22.10` container. From

> there, make sure you have installed `curl`. `telnet` is used by the install script
> to confirm the required port on the Teleport cluster is available:

Reviewer 3: A suggestion to tighten up the phrasing.

```suggestion
there, make sure you have`curl` and `telnet` installed. Both are used by the install script.
```

there, make sure you have installed `curl`:

```code

> $ apt update && apt install -y curl telnet

Reviewer 4: I'm not seeing the value here. Why are we asking the user to install a tool that we never mention again in the rest of the guide?

Reviewer: I think the idea is that the automatic join script can use it, but only if it's installed: https://github.com/gravitational/teleport/blob/master/lib/web/scripts/node-join/install.sh#L267-L269

Author: I added a description on its usage
